Assignment/solution_student/css/style.css

```diff
@@ -31,19 +31,120 @@ html {
 /* BASE STYLES
 /*******************************************/
 body {
-  color: black;
+  color: #121212;
   font-size: 16px;
-  background: white;
+  background-color: #f0efef;
+  font-weight: 300px;
```

Comment:
The font-weights are kinda weird, but they can be defined by a keyword like `bold`, or a unit-less number value in increments of 100 up to 900, so no `px` after, just `300`: https://developer.mozilla.org/en-US/docs/Web/CSS/font-weight